pom.xml

```diff
@@ -432,6 +432,35 @@
       </properties>
     </profile>
 
+    <!-- Copy newly built native libraries back to source tree -->
```

Comment:
You may -- if you wish -- add this target to `build.xml`.

e.g. https://github.com/java-native/jssc/blob/ea2ade2b3cbbc417ef9dfd97922a77fde41b911a/pom.xml#L213-L224